ooni/nodes.conf

```diff
@@ -1,53 +0,0 @@
-[control]
```

Comment:
This is some leftover from super old legacy code that had the notion of remotely administrable ooniprobe nodes that would be used for running tests on.

No code using it exists anymore and we don't have this type of usecase in mind anymore.